beets/autotag/match.py

```diff
@@ -78,7 +78,7 @@ class Recommendation(OrderedEnum):
 
 
 class Proposal(NamedTuple):
-    candidates: Sequence[AlbumMatch | TrackMatch]
+    candidates: Sequence[Union[AlbumMatch, TrackMatch]]
```

Comment:
Forgot to mention - there's no need for this adjustment as it already utilizes the new pipe syntax.

Reply:
it doesn‘t work below 3.10 though

Reply:
Try including `from __future__ import annotations` import at the top of the module